pkg/operator/config_reloader.go

```diff
@@ -230,6 +260,30 @@ func CreateConfigReloader(name string, options ...ReloaderOption) v1.Container {
 
 	boolFalse := false
 	boolTrue := true
+	if !configReloader.runOnce && configReloader.config.IsProbes {
+		return v1.Container{
+			Name:                     name,
+			Image:                    configReloader.config.Image,
+			ImagePullPolicy:          configReloader.imagePullPolicy,
+			TerminationMessagePolicy: v1.TerminationMessageFallbackToLogsOnError,
+			Env:                      envVars,
+			Command:                  []string{"/bin/prometheus-config-reloader"},
+			Args:                     args,
+			Ports:                    ports,
+			VolumeMounts:             configReloader.volumeMounts,
+			Resources:                resources,
+			LivenessProbe:            livenessProbe,
+			ReadinessProbe:           readinessProbe,
+			SecurityContext: &v1.SecurityContext{
+				AllowPrivilegeEscalation: &boolFalse,
+				ReadOnlyRootFilesystem:   &boolTrue,
+				Capabilities: &v1.Capabilities{
+					Drop: []v1.Capability{"ALL"},
+				},
+			},
+		}
```

Comment:
we can avoid redefining (duplication) the container by following the pattern above.

Reply:
done